core/spec.md

```diff
@@ -6710,29 +6566,39 @@ Where:
       `ATTRIBUTE` with this `VALUE` MUST be included in the response.
     - When `VALUE` is absent then the implied `VALUE` is an empty string and
       the matching MUST be done as specified in the previous bullet.
-  - `!=` operator:
+  - `!=`, `<>` operators:
     - When `VALUE` is `null` then it MUST have the same semantics as
      `?filter=ATTRIBUTE` as specified above (present with any non-`null`
      value).
     - When `VALUE` is non-`null` then only entities without the specified
       `ATTRIBUTE` and `VALUE` MUST be included in the response. This MUST be
       semantically equivalent to `NOT(ATTRIBUTE=VALUE)`, and this also means
       that if `ATTRIBUTE` is missing then that attribute will match the filter.
-
-When comparing an `ATTRIBUTE` to the specified `VALUE` the following rules
-MUST apply for an entity to be considered a match of the filter expression:
-- For boolean attributes, its value MUST be an exact case-sensitive match
-  (`true` or `false`).
-- For numeric attributes, it MUST be an exact match.
-- For string attributes, it MUST be an exact match (case-insensitive) when no
-  wildcard (`*`) character appears in the `VALUE`. The presence of a wildcard
-  indicates that any number of characters can appear at that location in the
-  `VALUE`. The wildcard MAY be escaped via the use of a backslash (`\\`)
-  character (e.g. `abc\*def`) to mean that the `*` is to be interpreted as
-  a normal character and not as a wildcard. Note that a `VALUE` of `*` MUST
-  be equivalent to checking for the existence of the attribute, with any
-  value (even an empty string). In other words, the filter will only fail if
-  the attribute has no value at all.
+  - `<`, `<=`, `>`, `>=` operators:
+    - `VALUE` MUST NOT be `null`.
+    - Only entities with the specified `ATTRIBUTE` and `VALUE` that matches
+      the comparison operator's semantics MUST be included in the response.
+    - `<` refers to "less than".
+    - `<=` refers to "less than or equal to".
+    - `>` refers to "greater than".
+    - `>=` refers to "greater than or equal to".
+    - Wildcards (`*`) (see below) MUST NOT be present in the `VALUE`.
+    - MUST only be used on numeric or string attribute types.
+
+When comparing an `ATTRIBUTE` to the specified `VALUE`, the type of the
+attribute impacts how the comparisons are done:
+- For boolean attributes, a successful `=` comparison MUST be an exact
+  case-sensitive match (`true` or `false`).
+- For numeric attributes, standard numeric comparisons rules apply.
+- For string attributes, the strings MUST be compared in a case-insensitive
```

Comment:
String *values* can be any Unicode, I assume? In which case we need to be more specific about comparisons. Ordinal, culture-insensitive, something else?

Reply:
I'd need to know what we actually want to happen first :)